src/main/java/org/folio/qm/client/MappingMetadataClient.java

```diff
@@ -0,0 +1,15 @@
+package org.folio.qm.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient("mapping-metadata")
+public interface MappingMetadataClient {
+
+  @GetMapping(value = "/type/{recordType}")
+  MappingMetadata getMappingMetadata(@PathVariable("recordType") String recordType);
+
+  record MappingMetadata(String mappingRules, String mappingParams) {
```

Comment:
```suggestion
  record MappingMetadata(String mappingRules, MappingParameters mappingParams) {
```
Class from di-processing-core could be used there.